Tasks/DanilaGromila(Melnik)/task1.py

```diff
@@ -0,0 +1,9 @@
+import decimal
+import math
+Deposit = 20000#депозит
+term = 5 #срок
+p = 0.15#процент
+month = term*12
+bank=decimal.Decimal(p) / decimal.Decimal(12)
+S = Deposit * (1 + bank)**month
+print(int(S))
```

Comment:
consider rounding to a second decimal digit, since our currency has a fraction to it ("kopeiki")